contracts/MemeFactory.sol

```diff
@@ -59,12 +62,12 @@ interface IERC20 {
 ///           $SMTH to be burned.
 abstract contract MemeFactory {
     event OLASJourneyToAscendance(uint256 amount);
-    event Summoned(address indexed summoner, uint256 indexed memeNonce, uint256 nativeTokenContributed);
+    event Summoned(address indexed summoner, uint256 indexed memeNonce, uint256 amount);
     event Hearted(address indexed hearter, uint256 indexed memeNonce, uint256 amount);
-    event Unleashed(address indexed unleasher, address indexed memeToken, uint256 indexed lpTokenId,
-        uint256 liquidity, uint256  nativeAmountForOLASBurn);
+    event Unleashed(address indexed unleasher, uint256 indexed memeNonce, address indexed memeToken, uint256 lpTokenId,
+        uint256 liquidity);
```

Comment:
Due to how the uniV3 dust works, it makes no sense returning this value. The on-chain value will be different potentially, just causing for confusion...